src/flows/context/api.tsx

```diff
@@ -1,28 +1,27 @@
 import {
-  postFlowsOrgsFlow,
-  PostFlowsOrgsFlowParams,
-  PatchFlowsOrgsFlowParams,
-  patchFlowsOrgsFlow,
-  deleteFlowsOrgsFlow,
-  getFlowsOrgsFlows,
-  DeleteFlowsOrgsFlowParams,
-} from 'src/client/flowsRoutes'
+  PatchNotebookParams,
+  patchNotebook,
+  PostNotebookParams,
+  postNotebook,
+  DeleteNotebookParams,
+  deleteNotebook,
+  getNotebooks,
+} from 'src/client/notebooksRoutes'
 import {notebookUpdateFail} from 'src/shared/copy/notifications'
 import {notify} from 'src/shared/actions/notifications'
 
-const DEFAULT_API_FLOW: PatchFlowsOrgsFlowParams = {
+const DEFAULT_API_FLOW: PatchNotebookParams = {
   id: '',
-  orgID: '',
   data: {},
 }
-let stagedFlow: PatchFlowsOrgsFlowParams = DEFAULT_API_FLOW
+let stagedFlow: PatchNotebookParams = DEFAULT_API_FLOW
 let reportDecayTimeout = null
 let reportMaxTimeout = null
 
 const REPORT_DECAY = 500 // number of miliseconds to wait after last event before sending
 const REPORT_MAX_WAIT = 5000 // max number of miliseconds to wait between sends
 
-export const pooledUpdateAPI = (flow: PatchFlowsOrgsFlowParams) => {
+export const pooledUpdateAPI = (flow: PatchNotebookParams) => {
```

Comment:
NIT: If we are getting away from flow(s) then shouldn't we rename these vars and CONSTANTS to notebook(s)?

Reply:
This is absolutely a fair point. But it's one of those things that if I start going down this train, there is soooooo much to refactor and rename 😭 I think it's an exercise we should do, but I think I'd like to do that in a separate PR.

Reply:
that was the same thinking that changed them all from notebooks to flows in the first place and broke things along the way. ROI seems... low. maybe i'm misunderstanding